src/engraving/libmscore/chord.cpp

```diff
@@ -1295,6 +1295,10 @@ bool Chord::readProperties(XmlReader& e)
     } else {
         return false;
     }
+    // Reset horizontal offset of grace notes when migrating from before 4.0
+    if (isGrace() && score()->mscVersion() < 400) {
+        rxoffset() = 0;
+    }
```

Comment:
I think this should be in the `read` method, because this `readProperties` method is called in a loop, basically once for every xml tag inside the <Chord> tag, and thus not all properties are initialized yet at the bottom of the `readPropeties` method (only one property is initialized per call to this method).

Reply:
Ha! You're totally right. Gonna do that now.